package-lock.json

```diff
@@ -2463,19 +2463,6 @@
         "react-dom": ">=16.8.0"
       }
     },
-    "node_modules/@dnd-kit/modifiers": {
```

Comment:
As mentioned @Olenka-Hryk, in my opinion it's better to commit without deleting some modules in packaga-lock.json. I know that this can appear, when you are installing your project by "npm i", but try with npm ci.

Or just try to add single file with your styles and commit it.

Reply:
got you guys. thanks. will revert changes in the package-lock json

Reply:
Looking for your updates

Reply:
@Olenka-Hryk @matviichuk-y followed your advices. waiting for your review ( :